client/cz-component-dash.html

```diff
@@ -0,0 +1,102 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../bower_components/paper-item/paper-item.html">
+<link rel="import" href="../bower_components/paper-item/paper-item-body.html">
+<script src="issues-lib.js"></script>
+
+<dom-module id="cz-component-dash">
+
+  <template>
+    <style>
+
+      paper-card .card-flex {
+        display: flex;
+        flex-wrap: wrap;
+        align-content: flex-start;
+      }
+
+      .component {
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      paper-item {
+        box-sizing: border-box;
+      }
+    </style>
+    <template is="dom-repeat" items="{{components}}">
+      <paper-card heading="{{item.cardHeading}}" id='card'>
+        <div class="card-content">
+          <div class='card-flex'>
+            <paper-item>
+              <paper-item-body>
+                <div secondary inner-h-t-m-l="[[issues(item.issues, updateSLO)]]" class='component'></div>
+              </paper-item-body>
+            </paper-item>
+          </div>
+        </div>
+      </paper-card>
+    </template>
+  </template>
+
+  <script>
+    Polymer({
+      is: "cz-component-dash",
+
+      properties: {
+        components: {
+          type: 'Object',
+          value: function() { return []; },
+          observer: "componentsChanged"
+        },
+        updateSLO: {
+          type: 'Object',
+          value: function() { return {}; },
+        }
+      },
+
+      attached: function() {
+        registerSource('cz-config', 'components', function(components) {
+          this.set('components', components);
+        }.bind(this));
+        registerSource('cz-config', 'updateSLO', function(updateSLO) {
+          this.set('updateSLO', updateSLO);
+        }.bind(this));
```

Comment:
It's probably simpler to put components and updateSLO into the same object in the config, and just issue 1 registerSource on 'cz-config'.